AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DirectionalLightAdded.py

```diff
@@ -146,37 +325,452 @@ def AtomEditorComponents_DirectionalLight_AddedToEntity():
+        Report.result(
+            Tests.fullscreen_blur_strength_set_to_default, Math_IsClose(
+                directional_light_component.get_component_property_value(
+                    AtomComponentProperties.directional_light('Fullscreen Blur Strength')), 0.667))
+
+        # 33. Fullscreen Blur Sharpness can be set to 240.0 then back to the default of 2.5.
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light('Fullscreen Blur Sharpness'), 240.0)
+        Report.result(
+            Tests.fullscreen_blur_sharpness_set_to_240, Math_IsClose(
+                directional_light_component.get_component_property_value(
+                    AtomComponentProperties.directional_light('Fullscreen Blur Sharpness')), 240.0))
+
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light('Fullscreen Blur Sharpness'), 50.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.fullscreen_blur_sharpness_set_to_default, Math_IsClose(
+                directional_light_component.get_component_property_value(
+                    AtomComponentProperties.directional_light('Fullscreen Blur Sharpness')), 50.0))
+
+        # 34. Turn Affects GI Blur off then back on.
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light('Affects GI'), False)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.affects_gi_off,
+            directional_light_component.get_component_property_value(
+                AtomComponentProperties.directional_light('Affects GI')) is False)
+
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light('Affects GI'), True)
+        Report.result(
+            Tests.affects_gi_back_on,
+            directional_light_component.get_component_property_value(
+                AtomComponentProperties.directional_light('Affects GI')) is True)
+
+        # 35. Factor can be set to 2.0 then back to the default of 1.0.
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light('Factor'), 2.0)
+        Report.result(
+            Tests.factor_set_to_2, Math_IsClose(
+                directional_light_component.get_component_property_value(
+                    AtomComponentProperties.directional_light('Factor')), 2.0))
+
+        directional_light_component.set_component_property_value(
+            AtomComponentProperties.directional_light('Factor'), 1.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.factor_set_to_default, Math_IsClose(
+                directional_light_component.get_component_property_value(
+                    AtomComponentProperties.directional_light('Factor')), 1.0))
+
+        # 36. Delete DirectionalLight entity.
```

Comment:
nitpick: should be "Directional Light" as that is the name property for the `directional_light` component

Reply:
fixed